plugins/collections/get.js

```diff
@@ -7,42 +7,104 @@ const getSchema = schema.models.collection.get;
 const idSchema = joi.reach(schema.models.collection.base, 'id');
 
 /**
- * Helper function to get last builds of pipelines
+ * Helper function to get PR info of pipeline
  *
- * @param   {Array}    pipelines     Array of pipeline models to get last builds for
- * @param   {Factory}  eventFactory  Factory for getting last event model
+ * @param   {Pipeline} pipeline  Pipeline model to get PR info for
+ * @returns {Object}
+ */
+function getPipelinePRInfo(pipeline) {
+    const prs = {
+        open: 0,
+        failing: 0
+    };
+
+    // Get all the PR jobs
+    return pipeline.getJobs({ type: 'pr' })
+        .then(prJobs => (
+            // Return array of prJobs' builds
+            Promise.all(prJobs.map(job =>
+                job.getBuilds()
+            ))
+        ))
+        .then((prJobsBuilds) => {
+            // The number of elements is the number of prs open
+            prs.open = prJobsBuilds.length;
```

Comment:
Is this true? 
I have 1 PR opened, and it has 5 builds. But prs.open should just be `1` not `5`

Reply:
Sorry I actually didn't know how to name it better but prJobsBuilds is a 2D array. Each element in prJobsBuilds is an array of builds for each of the PR Jobs